src/transaction_hash.rs

```diff
@@ -162,15 +160,17 @@ pub fn validate_transaction_hash(
     Ok(possible_hashes.contains(&expected_hash))
 }
 
-pub(crate) fn ascii_as_felt(ascii_str: &str) -> Result<StarkFelt, StarknetApiError> {
-    StarkFelt::try_from(hex::encode(ascii_str).as_str())
+// TODO: should be part of core::Felt
```

Comment:
Should it be left there?

Reply:
It is a valid todo. But it requires changes in the types-rs first.